RELEASING.adoc

```diff
@@ -0,0 +1,12 @@
+= Spring Cloud GCP release process
```

Comment:
I think there should be here steps that require the change of the version of the project from `SNAPSHOT` to release, and also changing it back to `SNAPSHOT` after the release from master.

Reply:
Added another step for this in latest commit.